scripts/qa_crowbarsetup.sh

```diff
@@ -3884,7 +3884,9 @@ function oncontroller_testsetup
         complain 95 could not reach internet
     fi
 
-    if iscloudver 7plus ; then
+    if iscloudver 7plus && crowbar nova proposal show default | rubyjsonparse \
```

Comment:
Please leave a short comment in the code about this very awkward condition check..

I'm agree with @bwiedemann that it would be nicer to just update the proposal on upgrade or remember that this is a cloud6 upgrade instead.